docker/owlbot/java/container_test.yaml

```diff
@@ -23,4 +23,4 @@ commandTests:
   expectedError: ["google-java-format: Version 1.7"]
 - name: "python"
   command: ["python", "--version"]
-  expectedOutput: ["Python 3.6.1"]
+  expectedOutput: ["Python 3.9.13"]
```

Comment:
@lqiu96 Would you add an assertion that this python module `synthtool.languages.java` is available? I think it would look like this:

```
python3 -c 'import synthtool.languages.java; print("import successs")'
```

An example failure would look like this:

```
suztomo@suztomo:~/java-storage$ docker run --user "$(id -u):$(id -g)" --entrypoint /bin/bash --rm -it  a8e0ab6765a4
I have no name!@752c3d65c2dd:/workspace$ python3 -c 'import synthtool.languages.java; print("import successs")'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
ModuleNotFoundError: No module named 'synthtool.languages'
I have no name!@752c3d65c2dd:/workspace$ 
```

It gives more confidence that a change for OwlBot Postprocessor works fine in owlbot.py ([example location](https://github.com/googleapis/google-cloud-java/blob/main/java-accessapproval/owlbot.py#L16))

Reply:
Sure, I can add that validation in

Reply:
Added the new validation in as `java synthtool validation` which runs the commands above.

FYI, couldn't find the docs for `schemaVersion` 1.0.0 and the docs on https://github.com/GoogleContainerTools/container-structure-test were showing docs for `2.0.0`, so I used that for local testing

Reply:
Thank you.